docs/contributing-to-analyses/pr-review-and-merge/respond-to-review.md

```diff
@@ -0,0 +1,171 @@
+
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-03.png){width="600"}
+</figure>
+
+You can directly reply to these review comments on GitHub.
+For example, you might ask your reviewer to clarify a particular comment, or ask for more guidance about how to implement the review.
+Or, you might just want to leave your reviewer a helpful note of some kind!
+
+The way to responding to an existing comment differs depending on which PR tab you are leaving the comment from:
+
+- If you are in the **`Files changed`** tab, use the button `Add single comment` to reply (_do not use `Start a review`_):
+    <figure markdown="span">
+        ![Reply to a review comment from the `Files changed` tab.](../../img/respond-to-review-04.png){width="600"}
+    </figure>
+
+- If you are in the **`Conversation`** tab, use the button `Comment`.
+    <figure markdown="span">
+        ![Reply to a review comment from the `Conversation` tab.](../../img/respond-to-review-05.png){width="600"}
+    </figure>
+
+
+### Suggestion comments
+
+Reviewer comments can also include _suggestions_ for code changes, that you can incorporate into your PR directly on GitHub.
+
+For example, below are two suggestions left as comments on an R script:
+
+- In the first comment, the reviewer suggested that the code author add a new line to load the `readr` library.
+- In the second comment, the reviewer suggested that the code author remove a line that loads the `dplyr` library.
+
+
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-06.png){width="600"}
+</figure>
+
+
+Suggestion comments have two associated buttons boxed in the image above: `Commit suggestion` and `Add suggestion to batch`.
+
+- You can directly commit a _single_ suggestion by clicking `Commit suggestion` button, and then clicking `Commit changes` in the dropdown box:
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-07.png){width="600"}
+</figure>
+
+- Alternatively, if there are multiple suggestions to accept, you can click the `Add suggestion to batch` button on each one.
+_The `Add suggestion to batch` option is only available via the `Files changed` tab!_
+    - Now, both suggestions are `Pending in batch` to be committed as a group:
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-08.png){width="600"}
+</figure>
+
+You can commit the batched suggestions at once with the `Commit suggestions` button that will appear at the top of the `Files changed` tab:
+
+<figure markdown="span">
+    ![Re-request review.](../../img/respond-to-review-09.png){width="600"}
+</figure>
+
+
+Read this GitHub documentation for more about [applying suggested changes while responding to review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes).
+
+
+
+## Responding to review comments
```

Comment:
```suggestion
## Making additional code changes in response to comments
```
What do we think of this for this section? This is really all about making changes in response to additional comments that are not just direct suggestions.